### DISCUSSION TITLE
How to get this library discovered by more people?

### DISCUSSION THREAD

**Discussion by vestrel00** [vestrel00/contacts-android#159] [General]
While I truly do believe that a good product/software will sell itself, it can't be sold if people don't even know that it is on sale 😭 

This is a stark realization that I've come to terms with. No matter how hard I try and how hard I work, my efforts will remain in a bubble unless I "spread the word" or "self-promote". It does not matter if I spend 40+ hours per week on this, perfecting it. I'm not building this library for myself to look at. I genuinely want the android community to use it because I think the 3+ years that I've spent on this is worthy of integrating into their project.

I have removed myself from all social media.

- I haven't been on Facebook for almost a decade now.
- I have a Twitter account from 2013 with 0 tweets; https://twitter.com/vestrel00
- I have a Reddit account that I just created 2 months ago with 0 posts; https://www.reddit.com/user/vestrel00

Staying out of social media has done wonders for my health and I have no regrets. However, I did not know how difficult it would be to share this passion project of mine without it.

I don't have the social media presence that seems like is a requirement to get stars, or at least discoverability, on a repo. It's not that I'm "star hungry". I honestly could not care less about stars. However, it does not change the fact that stars are the most common way for most people to choose which library to use. Typically, people just use the one that has more stars because it is so easy to decide that way. Even I am guilty of this myself, sometimes.

### What are your thoughts?

What do you think? Do you think that I should just put my head down and work on this and forget about "spreading the word"? Are stars really a measure of a project's usefulness and potential? Should I not care about stars?

**Comment by amoshalava** [comment]
This is a great project! Promote it as much as you can. It isn't about fame as much as making a difference. Also create a way for people to donate to you so that it can pay for its self.

**Reply by vestrel00** [reply]
I agree that this isn't about fame. That's not what I'm after. I'm also not after sponsorships. I only want to make sure that this repo is discoverable by everyone =)

**Comment by vestrel00** [comment]
I finally got around to promoting this library in r/androiddev a week ago. It was a big success!

https://www.reddit.com/r/androiddev/comments/rz370s/i_dedicated_3_years_to_building_this_android

I really did not expect it to be received so well by the Reddit community. To all of those from Reddit that are reading this. Thank you! 😁 ❤️
